src/nvim/api/ui.c

```diff
@@ -340,7 +345,56 @@ void nvim_ui_pum_set_height(uint64_t channel_id, Integer height, Error *err)
                   "It must support the ext_popupmenu option");
     return;
   }
-  ui->pum_height = (int)height;
+
+  ui->pum_nlines = (int)height;
+}
+
+/// Tells Nvim the geometry of the popumenu, to align floating
```

Comment:
Normally a docstring begins with a single-line header (though perhaps two-lines is ok, as used above).

Reply:
Do you mean the help file? I split it into two lines because it doesn't fit into one. There are a few other functions with long signature that takes this two-line form so I adapted that.

Reply:
I mean this text here that I commented at. There is no header line here.

Reply:
I see. Thanks for the clarification. Updated.